### PR TITLE
Recruitifyv1

### PR DESCRIPTION
**Pull request by Tarik-Anowar**

### CODE REVIEW

File: backend/index.js
Comment (priyam-03) [inline backend/index.js:1]: Please make it same as of main branch index.js file

File: frontend/package.json
Comment (priyam-03) [inline frontend/package.json:1]: Keep it http only

**Review by priyam-03 — CHANGES_REQUESTED** [review]
Please make these changes and commit.

File: frontend/package.json
Comment (Tarik-Anowar) [reply frontend/package.json:1]: ok

**Comment by Tarik-Anowar** [comment]
updated the Home Page and some post related backend stuffs

**Comment by priyam-03** [comment]
Home page updated.

**Comment by priyam-03** [comment]
Home page added.

**Pull request merged**